chanjo/cli/init.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 import codecs
-from distutils.spawn import find_executable
```

Comment:
Googling like crazy for reviewing this PR. This seems to be deprecated in python 3.12. Nice with the replacement! 👍🏻